ir/ir.def

```diff
@@ -205,6 +205,25 @@ class Key : PropertyValue {
     validate { keyElements->check_null(); }
 }
 
+/// Pre-defined entry in a table
+class Entry : IAnnotated {
+    Annotations         annotations;  /// annotations are optional (supported annotations: @priority(value))
+    ListExpression      keys;         /// must be a tuple expression
+    ActionListElement   action;       /// action must be defined in action list
```

Comment:
This should also be an expression; in general it will be a MethodCallExpression.
Don't use ActionListElement, there control-plane arguments are not bound, so the type-checking is different.